src/e2e-test/features/pubsub/pubsubadditionalfeature/PubSubToPubSub.feature

```diff
@@ -0,0 +1,166 @@
+      Then Save the pipeline
+      Then Deploy the pipeline
+      Then Enter runtime argument value for PubSub source property topic key "PubSubSourceTopic"
+      Then Enter runtime argument value for PubSub source property subscription key "PubSubSourceSubscription"
+      Then Run the Pipeline in Runtime with runtime arguments
+      Then Wait for pipeline to be in status: "Running" with a timeout of 240 seconds
+      Then Publish the messages
+      Then Subscribe to the messages
+      Then Validate OUT record count is equal to IN record count
+      And Stop the pipeline
+      Then Verify the pipeline status is "Stopped"
+
+  @PUBSUB_SOURCE_TEST @PUBSUB_SINK_TEST @PUBSUB_SUBSCRIPTION_TEST @PUBSUB_MESSAGE_TEST
+  Scenario: Validate successful transfer of records from PubSub(source) to PubSub(sink)  having formats TEXT at both source and sink.
+    Given Open Datafusion Project to configure pipeline
+    When Select data pipeline type as: "Realtime"
+    When Expand Plugin group in the LHS plugins list: "Source"
+    When Select plugin: "Pub/Sub" from the plugins list as: "Source"
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "Pub/Sub" from the plugins list as: "Sink"
+    Then Connect plugins: "Pub/Sub" and "Pub/Sub2" to establish connection
+    Then Navigate to the properties page of plugin: "Pub/Sub"
+    Then Override Service account details if set in environment variables
+    Then Enter input plugin property: "referenceName" with value: "BQReferenceName"
+    Then Enter PubSub source property subscription name
+    Then Enter PubSub source property topic name
+    Then Select dropdown plugin property: "select-format" with option value: "text"
+    Then Validate "Pub/Sub" plugin properties
+    And Close the Plugin Properties page
+    Then Open the PubSub sink properties
+    Then Enter PubSub property projectId "projectId"
+    Then Override Service account details if set in environment variables
+    Then Enter PubSub property reference name
+    Then Enter PubSub sink property topic name
+    Then Select dropdown plugin property: "select-format" with option value: "text"
+    Then Validate "Pub/Sub" plugin properties
+    And Close the Plugin Properties page
+    And Click on configure button
+    And Click on pipeline config
+    And Click on batch time and select format
+    Then Save the pipeline
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait for pipeline to be in status: "Running" with a timeout of 240 seconds
+    Then Publish the messages
+    Then Subscribe to the messages
+    Then Validate OUT record count is equal to IN record count
+    And Stop the pipeline
+    Then Verify the pipeline status is "Stopped"
+
+  @PUBSUB_SOURCE_TEST @PUBSUB_SINK_TEST @PUBSUB_SUBSCRIPTION_TEST @PUBSUB_MESSAGE_TEST
+  Scenario: Validate successful transfer of records from PubSub(source) to PubSub(sink)  having formats TEXT at source and JSON at sink.
```

Comment:
Validate the successful transfer of records from a pubSub source to a pubSub sink with format Text at source and Json at sink.

Reply:
Changed to above